src/icon/avatarIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react'
+import cc from 'classcat'
+import isEmpty from 'lodash.isempty'
+
+import { color } from '_utils/branding'
+
+class AvatarIcon extends PureComponent<Icon> {
+  static defaultProps: Icon = {
+    className: '',
+    iconColor: color.icon,
+    size: 24,
+    title: '',
+  }
+
+  render() {
+    const { className, iconColor, size, title } = this.props
+    return (
+      <svg
+        viewBox="0 0 19 20"
```

Comment:
viewbox 19?

Reply:
All these icons have different sizes... We already have some specificities like this :(

Reply:
Search `viewBox=` and see the thruth